### PR TITLE
added pd as namespace

### PR DESCRIPTION
**Pull request by dacoex**
more consistent with code in the rest of the document

### CODE REVIEW

File: doc/source/io.rst [hunk 1]
```diff
@@ -2801,6 +2801,24 @@ everything in the sub-store and BELOW, so be *careful*.
 
 .. _io.hdf5-types:
 
+.. warning:: Hierarchical keys cannot be retrieved as dotted (attribute) access as described above for items stored under root node.
```
Comment (shoyer) [inline doc/source/io.rst]: This belongs in a separate PR --- or I would consider renaming this one to make it obvious that this is the big change here.

Comment (dacoex) [inline doc/source/io.rst]: Either way, what would you prefer?

Comment (shoyer) [inline doc/source/io.rst]: I'd opt for another PR, it just keeps things easier if PRs are smaller and about one thing only.

**Comment by jreback** [comment]
@dacoex yeh, you will need to explain why you want to add a warning like that (in another PR).

**Comment by dacoex** [comment]
@jreback seems that you are not convinced of the usefulness.

**Comment by jreback** [comment]
I am talking about the keys accessor warning. just take out for now and you can put in another PR. (e.g. just the ``pd.`` changes here.

No I am not convinced its something people should be doing at all. The attribute access is just a convenience feature, string accessing by paths is the preferred method.

**Comment by dacoex** [comment]
OK, now this PR is only about the namespace changes.

**Comment by jreback** [comment]
can you rebase this

File: doc/source/io.rst [hunk 2]
```diff
@@ -2801,6 +2801,22 @@ everything in the sub-store and BELOW, so be *careful*.
...
+
+    .. ipython:: python
+    
+       store.foo.bar.bah
```
Comment (jreback) [inline doc/source/io.rst]: this sectionneeds to be in a separete PR

Comment (dacoex) [inline doc/source/io.rst]: I thought with was already merged. So here you go.

**Comment by jreback** [comment]
pls don't open new pull-requests for the same fix, just force push to the same. see the [docs](http://pandas.pydata.org/pandas-docs/stable/contributing.html)

simply

``git push yourbranch yourremote -f``
